api/Dockerfile

```diff
@@ -38,4 +38,4 @@ EXPOSE 8080
 EXPOSE 8081
 ENV LOG_LEVEL Debug
 # ENV CONFIG /symphony-api.json
-CMD exec /symphony-api -c $CONFIG -l $LOG_LEVEL
\ No newline at end of file
+CMD sh -c 'if [ -f /etc/pki/ca-trust/source/anchors/proxy-cert.crt ]; then update-ca-trust; fi && exec /symphony-api -c $CONFIG -l $LOG_LEVEL'
```

Comment:
Can we log an issue to update the doc on how certificates are configured|? Similar to Azure proxy configuration.

Reply:
https://github.com/eclipse-symphony/symphony/issues/239